module-4/backend/week_two.md

```diff
@@ -4,8 +4,59 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-TBD  
+1. What's one difference between ES5 and ES6?
+
+ES6 is moving JS to more natively support OOP.  One instance of this is the new Class structure.
+
+2. What's the difference between asynchronous and synchronous JavaScript? 
+
+synchonous JS runs all at once where as asynchronous allows us to tell the program to run Y line of code after X has happened successfully.
+
+3. What are the four pillars of Object Oriented programming?
+
+Abstraction - only show the outside world what is needed for it to interact with an object.  
+Polymorphism - can redefine functionality by reusing a name  
+Inheritance - create and share code through relationships between classes  
+Encapsulation - bind data and functions together in one class  
+
+4. What are some tools available in JavaScript to help you write object oriented code?
+
+`Class`es `static` methods and instance methods.
```

Comment:
Keep in mind that "instance" methods in JS are more accurately referred to as prototype functions.